driver/src/main/java/org/neo4j/driver/internal/handlers/InitResponseHandler.java

```diff
@@ -71,11 +72,25 @@ public void onRecord( Value[] fields )
         throw new UnsupportedOperationException();
     }
 
-    private static ServerVersion extractServerVersion( Map<String,Value> metadata )
+    private static ServerVersion extractServerVersion( Map<String,Value> metadata ) throws UntrustedServerException
```

Comment:
`throws` can be removed because `UntrustedServerException` is a runtime exception